### ISSUE TITLE
Fresh install problem

### ISSUE THREAD

**Issue by madzat** [AdKats/BFACP#280]
Hi i just want to try today the bfacp and i want to ask if is working with xampp 

the error i have all time is 
 Database Error!
Sorry, something went wrong with the database.
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'bf4.bfacp_permissions' doesn't exist (SQL: select * from `bfacp_permissions`)

any help please

**Comment by Prophet731** [comment]
Yes it will work with xampp, as long as you have the software needed to run it it should work.
